001-update-metainfo.patch

```diff
@@ -0,0 +1,98 @@
+--- a/flatpak/tech.feliciano.pocket-casts.appdata.xml	2024-09-26 04:18:40.151643139 -0400
++++ b/flatpak/tech.feliciano.pocket-casts.appdata.xml	2024-09-26 04:19:23.269545738 -0400
+@@ -1,31 +1,80 @@
+ <?xml version="1.0" encoding="UTF-8"?>
+-<component type="desktop">
++<component type="desktop-application">
+   <id>tech.feliciano.pocket-casts</id>
+-  <launchable type="desktop-id">tech.feliciano.pocket-casts.desktop</launchable>
++
+   <name>Pocket Casts</name>
++  <summary>Linux desktop player for Pocket Casts</summary>
+   <developer_name>Ricardo N Feliciano</developer_name>
+-  <summary>Online podcast player</summary>
```

Comment:
Why the summary change?

Reply:
I initially was thinking I would change it to be more descriptive. Looking now: https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/quality-guidelines#summary

It may also be worth it to copy what is used for the snap for consistency's sake: https://github.com/FelicianoTech/pocket-casts-desktop-app/blob/06d0439647b500b0ab84566d2d9028d1cff5ede6/package.json#L45

Main issue with that is it may not work out to be totally within the quality guidelines.